### ISSUE TITLE
Improve bot post description 

### ISSUE THREAD

**Issue by AndrewBestbier** [bag-man/dm_bot#6]
I think that your bot would do a lot better if you fleshed out the post text a bit more. An example would be 'Non-daily mail link. Code on GitHub. Why is the bot built' and then you could link to another site or even GitHub page with your justifications. Currently other users have to explain to confused users in the comments. This is limiting your reach :)

**Comment by bag-man** [comment]
I had initially launched the bot with a description [like this](https://www.reddit.com/r/test/comments/2kbm6k/amal_clooney_channels_old_hollywood_glamour_in/cljpsin) as I felt it was important to raise awareness to the reasons behnd the bot. 

However I got a lot of feedback from users that it was too much text for one post, and that it felt like preaching spam. Perhaps there is a happy medium, but when I changed it to the current system people seemed a lot happier. By not adding extra info it does allow the bot to remain slightly less political and just a service that is provided. 

Perhaps there is a happy medium?

**Comment by AndrewBestbier** [comment]
Fair points all round - I agree that the previous version was a bit too wordy. Perhaps a happy medium could simply be the Non Daily Mail link and 'Why The Daily Mail is Evil'. Either way you choose, great bot idea and implementation :)
